src/main/java/com/spotify/reaper/storage/MemoryStorage.java

```diff
@@ -300,6 +304,31 @@ public boolean updateRepairSegment(RepairSegment newRepairSegment) {
   }
 
   @Override
+  public Collection<RepairParameters> getOngoingRepairsInCluster(String clusterName) {
```

Comment:
The nicest way I could come up with sadly was loops. The real answer is hopefully Java8. I think we should go there with the 2.1 support.